webpack.config.js

```diff
@@ -83,7 +83,7 @@ function lorisModule(mname, entries) {
         output: {
             path: path.resolve(__dirname, 'modules') + '/' + mname + '/js/',
             filename: '[name].js',
-            library: ['lorisjs', mname],
+            library: ['lorisjs', mname, '[name]'],
```

Comment:
This is required otherwise the last export always overwrites the whole module in the lorisjs object.